suite-native/module-accounts-import/src/components/AccountImportLoader.tsx

```diff
@@ -1,41 +1,76 @@
-import React from 'react';
+/* eslint-disable global-require */
+import React, { useEffect } from 'react';
+import { Image } from 'react-native';
+import Animated, {
+    useAnimatedStyle,
+    withDelay,
+    withSequence,
+    withTiming,
+} from 'react-native-reanimated';
 
-import { Box, Text } from '@suite-native/atoms';
+import { Box, Loader, Text, VStack } from '@suite-native/atoms';
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 
-const fetchingStyle = prepareNativeStyle(() => ({
+type AccountImportLoaderProps = {
+    onLoad(): void;
+};
+
+const loaderContainerStyle = prepareNativeStyle(utils => ({
     flex: 1,
-    justifyContent: 'center',
+    justifyContent: 'space-between',
     alignItems: 'center',
+    marginTop: utils.spacings.medium * 10,
```

Comment:
Why such a large margin? This won't work on different screen sizes too well. Why couldn't you use flex positioning? I assume it shouldn't be same margin on iPhone and iPad and should be responsive as much as possible.

Reply:
Because I need to move the image component to the centre of the screen. I could also solve it by putting there an extra empty component (invisible) and setting `justifyContent: 'space-between'. What do you think?

Reply:
Oh I see, I thought there were 3 elements.